lib/clients/git-client.js

```diff
@@ -36,12 +36,13 @@ module.exports = class GitClient {
     /**
      * Config local git credential helper with credentialHelperPath
      * @param {string} credentialHelperPath the path of git credential helper
+     * @param {string} hostUrl host url to apply credential helper to
```

Comment:
nit: extra `to` at the end of the sentence

Reply:
actually I meant to use "to",  it was not a typo.